editor/src/core/workers/common/project-file-utils.spec.ts

```diff
@@ -128,4 +128,20 @@ describe('mergeImports', () => {
       'component-library': importDetails(null, [importAlias('Card'), importAlias('FlexRow')], null),
     })
   })
+
+  it('handles multiple imports for the same file with different paths', () => {
+    const result = mergeImports(
+      '/src/code.js',
+      {
+        '/src/fileA.js': importDetails(null, [importAlias('Card')], null),
```

Comment:
Very minor, but please can you remove the `.js` suffixes from the import statements, since you wouldn't typically include those, and I just want to make sure we're not accidentally (now or in the future) depending on them

Reply:
I only did it like this because the entire rest of the file is like it.

Reply:
Fixed in 8c71e97c.